### ISSUE TITLE
Cannot create octane blueprint when behind a proxy (w/ (?) self-signed cert)

### ISSUE THREAD

**Issue by NullVoxPopuli** [ember-cli/ember-octane-blueprint#184]
https://discordapp.com/channels/480462759797063690/480462759797063692/624324303948611636
```
 - stack: RequestError: self signed certificate in certificate chain
    at ClientRequest.request.once.error (/tmp/ember-cli2019819-70394-1fv1j05.ggvjf/node_modules/package-json/node_modules/got/source/request-as-event-emitter.js:178:14)
    at Object.onceWrapper (events.js:286:20)
    at ClientRequest.emit (events.js:203:15)
    at ClientRequest.origin.emit.args (/tmp/ember-cli2019819-70394-1fv1j05.ggvjf/node_modules/@szmarczak/http-timer/source/index.js:37:11)
    at TLSSocket.socketErrorListener (_http_client.js:392:9)
    at TLSSocket.emit (events.js:198:13)
    at emitErrorNT (internal/streams/destroy.js:91:8)
    at emitErrorAndCloseNT (internal/streams/destroy.js:59:3)
    at process._tickCallback (internal/process/next_tick.js:63:19)
```

looks to be an issue with the `got` package, which we use for getting the latest URLs for things.

This will stop being an issue when we are no longer on git versions of things, and are using regular npm/yarn/published versions

**Comment by rwjblue** [comment]
Hmm, is there a flag or environment variable for `got` we could use?

**Comment by rwjblue** [comment]
Octane was released in Ember 3.15! This repo has been deprecated (the blueprints that come with ember-cli are the source of truth again). I'm closing as this repo is being archived, but if you are still running into issues please file a new issue over on ember-cli's issue tracker.

Thanks for your help making Octane happen!!
